web/docs/project/env-vars.md

```diff
@@ -6,14 +6,14 @@ title: Env Variables
 
 For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you may prefer it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
 
-While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes.
+While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes, and then access them in the code.
 
 In Wasp, you can use environment variables in both the client and the server code.
 ## Client Env Vars
 
-Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
+Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys -> you can provide those to server instead).
```

Comment:
Let's also get rid of the passive form (are embedded -> Wasp embeds).

I'm not a fan of using parentheses and an arrow here - it's a little weird, and I feel it could be it's own sentence:
```
For example, you shouldn't provide your API keys as client env vars. Instead, set them on the server.
```
Btw, I'm not sure this makes sense. The API keys will still be visible by anyone who accesses the web page (not storing them in the env file will just prevent them from getting into version control).

Reply:
Same as I said above, I would not try to improve the whole pieces of docs here in this PR, but would just focus on adding neccessary bits that I thoguht are important based on the user's experience. We can make a separate effort of improving the docs flow and sentencing, we will have better results if we do it as a whole, then if we attack only random bits that were touched in this PR, missing context.

I don't see much improvement with the sentence you provided. Also, you dropped word "secret", which is crucial -> they can provide public API keys.

I don't get what doesn't make sense to you? We are not mentioning env files at all here, how did you get to those?

Reply:
> We can make a separate effort of improving the docs flow and sentencing, we will have better results if we do it as a whole, then if we attack only random bits that were touched in this PR, missing context.

Ok, makes sense.

> I don't get what doesn't make sense to you?

I get it know, you're talking about some data that the client doesn't need at all (it's only required by the server). But why would they then pass it to the client in the first place if the client doesn't need them?

Reply:
So it is actually a quite common question/mistake we get in Discord: I have let's say Sendgrid API secret, can I put it in the client? So I don't have to bother with the server. I think you know quite a bit about it so it seems this is obvious, but you would be surprised.

Aha you say that client doesn't need them -> if they decide to use some third party service, let's say S3, it is their choice how to impelemnt it in Wasp, and they might easily try to do it all in client. With our features it is clear where env var goes, but if they are doing it alone like this, then they can more easily make a mistake.